### PR TITLE
Ensemble to it's own package

### LINKED ISSUE #2717: Ensemble project shouldn't be part of Microsoft.ML
From public perspective it's just dll with 0 public classes and none of other projects in Microsoft.ML rely on it.

### PR DESCRIPTION
**Pull request by Ivanidzo4ka**
fix #2717

### CODE REVIEW

File: src/Microsoft.ML.Ensemble/Microsoft.ML.Ensemble.csproj [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <IncludeInPackage>Microsoft.ML</IncludeInPackage>
```
Comment (eerhardt) [inline src/Microsoft.ML.Ensemble/Microsoft.ML.Ensemble.csproj]: Should this assembly go in a different nuget package?

Comment (eerhardt) [reply src/Microsoft.ML.Ensemble/Microsoft.ML.Ensemble.csproj]: I guess there aren't any public types in this assembly....?

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Ensemble/Microsoft.ML.Ensemble.csproj]: Do we have way to mark stable nugets vs experimental?
It's needed by NimbusML, but in same time, it feels weird to create nuget with zero public surface.

Comment (eerhardt) [reply src/Microsoft.ML.Ensemble/Microsoft.ML.Ensemble.csproj]: We don’t have it yet. But #2279 will enable it. For now let’s make the new package and we can mark it experimental when we have that.

Btw We are creating a Parquet package now, but we don’t ship it.

**Comment by Ivanidzo4ka** [comment]
We discuss and decided to leave it as it is for right now.

**Comment by eerhardt** [comment]
I don't think the way it is right now is correct. Why are we shipping this assembly in the core nuget package?

**Comment by TomFinley** [comment]
> We discuss and decided to leave it as it is for right now.

Sorry, who discussed? Where? I think we need a new nuget to capture this.

**Comment by eerhardt** [comment]
@Ivanidzo4ka - can you just keep this PR to be for `Ensemble`?  We can do the FastTree work separate.  There is also the native library that needs to move out of `Microsoft.ML`. I'd feel more comfortable to do Ensemble first. FastTree separate.

**Comment by Ivanidzo4ka** [comment]
Sure

---
In reply to: [467669272](https://github.com/dotnet/machinelearning/pull/2718#issuecomment-467669272) [](ancestors = 467669272)

File: Microsoft.ML.sln [hunk 2]
```diff
@@ -260,6 +260,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.Data.DataView", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RemoteExecutorConsoleApp", "test\RemoteExecutorConsoleApp\RemoteExecutorConsoleApp.csproj", "{5E920CAC-5A28-42FB-936E-49C472130953}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.ML.Ensemble", "Microsoft.ML.Ensemble", "{AD7058C9-5608-49A8-BE23-58C33A74EE91}"
+	ProjectSection(SolutionItems) = preProject
+		pkg\Microsoft.ML.Ensemble\Microsoft.ML.Ensemble.nupkgproj = pkg\Microsoft.ML.Ensemble\Microsoft.ML.Ensemble.nupkgproj
```
Comment (TomFinley) [inline Microsoft.ML.sln]: Not aware of us doing this elsewhere, but that's OK I guess.

**Pull request merged**